package.json

```diff
@@ -63,6 +64,7 @@
   "devDependencies": {
     "@jfhbrook/swears": "^1.0.2",
     "@nestjs/testing": "^10.3.10",
+    "@swc/cli": "^0.4.1-nightly.20240914",
```

Comment:
nightly?? lmaooo

Reply:
The swc CLI project seems... messy. 0.4.0 was published over a year ago, and this was released about a month ago - possibly as a nightly on accident? The npm page also says that `swxc` is ready for beta testing, but it's been [broken on mac for over a year](https://github.com/swc-project/swc/issues/4017#issuecomment-2392689754)? Either way, it seems like the swc project's energy has been largely spent on the API used by tools like vite, and the CLI tool has comparatively languished.

Either way, given that they're not constantly releasing nightlies, I think this is OK.